enchant.lic

```diff
@@ -59,14 +60,12 @@ class Enchant
     @equipment_manager = EquipmentManager.new
     @equipment_manager.empty_hands
 
-    if @use_own_brazier
-      walk_to(@settings.enchanting_room) if @settings.enchanting_room
-    end
-
     study_recipe
-
-    get_item(@fount)
-    bput("wave #{@fount} at #{@item} on #{@brazier}", 'You slowly wave')
+    @item = "fount" if @item == "small sphere"
+    #get_item(@fount)
+    fput("get my #{@fount} from my #{@bag}")
```

Comment:
❓ Should this use the crafting-specific 'get' function?

Reply:
`get_crafting_item` and `stow_crafting_item`

Reply:
I can change to that. I just used @dr-leustyin  script he created for ;craft and have had to modify a little for workorders.

Reply:
It does, get_item and stow_item in the script call those.